services/user/Evaluations/ui/src/components/new-eval.tsx

```diff
@@ -0,0 +1,225 @@
+import { useCreateEvaluation } from "@hooks/app/use-create-evaluation";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { useAppForm } from "./app-form";
+import { humanize } from "@lib/humanize";
+import { NumbersField } from "./numbers-field";
+
+dayjs.extend(duration);
+
+const lowestFifth = (minute: number) => minute - (minute % 5);
+
+const now = dayjs();
+
+const registration = now;
+const deliberation = now.add(10, "seconds");
+const submission = deliberation.add(30, "seconds");
+const finishBy = submission.add(10, "seconds");
```

Comment:
Can you set these defaults to something like 2 minutes for each phase?

Reply:
Done